realm/src/main/java/io/realm/internal/TableView.java

```diff
@@ -73,19 +73,24 @@
      * @param parent A table.
      * @param nativePtr pointer to table.
      */
-    protected TableView(Context context, Table parent, long nativePtr){
+    protected TableView(Context context, Table parent, long nativePtr) {
```

Comment:
Yes, the `io.realm.internal.Table` class has a few methods which return a `TableView` object with going through a query. An example is `getSortedView()`.